src/sentry/workflow_engine/models/incident_groupopenperiod.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class IncidentGroupOpenPeriod(DefaultFieldsModel):
+    """
+    A lookup model for incidents and group open periods.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    incident_id = BoundedBigIntegerField(null=True, unique=True)
```

Comment:
Should this be nullable? I guess it's null for incidents created after we stop writing to `Incident`?

Reply:
yeah nullable just in case we need to fudge the id